_posts/en/2021-09-22-po-free-developer-time-with-integromat.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: post
+title: "PO : Free some developer time by doing some NoCode with Integromat"
+excerpt: You're part of a small team with a lot of feature to release quickly and a PoC to test a product but you only have little to no tech skills ? NoCode is what you're looking for !
+authors:
+    - marianne
+permalink: /en/po-free-developer-time-with-integromat/
+categories:
+    - best practice
+    - nocode
+
+cover: /assets/2021-09-22-po-liberez-du-temps-a-vos-developpeurs-avec-integromat/cover.jpg
+---
+
+As a PO or product manager, you can develop yourself some features that may be repetitive and uninspiring for your developers, without much added value. This allows teams to focus on the essentials, while leaving more time available to think about the architecture and the various technical challenges. To test a new idea and/or market without investing too much, a NoCode tool is a real solution.
+
+I already showcased here open-source projet [n8n](https://blog.eleven-labs.com/fr/outil-low-code-automatisation-workflow-n8n/) (post in french), a tool dedicated to make Low Code automation. [Integromat](https://www.integromat.com/) aim to go even further with plain NoCode, thus even usable wihout any technical
+
+To try Integromat a freeplan is available to create your first scenarios. After that, affordable
+
+With first plans at affordable prices, it is easy to go from PoC to a professional use.
```

Comment:
C'est sur 2 lignes différentes, je me demande si le "After that, affordable" n'est pas à supprimer.

Reply:
Bien vu, j'ai reformulé